pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/WorkflowDao.java

```diff
@@ -64,6 +64,13 @@ default void insertTransitions(List<WorkflowTransition> transitions) {
         }
     }
 
+    default int deleteStudyWorkflow(long studyId) {
```

Comment:
Is this method called from `TransactionWrapper.useTxn`?  As it calls several update queries.

Reply:
Ah, yes it is. Just checked from code.